cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -539,7 +539,9 @@ def _create_block(request):
         # Paste from the user's clipboard (content_staging app clipboard, not browser clipboard) into 'usage_key':
         try:
             created_xblock, notices = import_staged_content_from_user_clipboard(
-                parent_key=usage_key, request=request
+                parent_key=usage_key,
+                request=request,
+                link_to_upstream=request.json.get("link_to_upstream"),
```

Comment:
I think what we discussed recently was that instead of prompting the user to ask if they want to link or not, we should just always assume `link_to_upstream=True` if pasting from a library, and authors can just break the link if they don't want it linked.

Reply:
Yup, you're right. Done.